src/manual/tasks/taskgroupid-schedulerid.md

```diff
@@ -0,0 +1,23 @@
+---
+filename: tasks/taskgroupid-schedulerid.md
+title: The taskGroupId and schedulerId properties
+order: 45
+---
+
+Each task is assigned to a task group by its `taskGroupId`. Task groups
+associate "related" tasks, such as those created in response to the same
+version-control push. The Queue service provides an API method to list all
+tasks with a specific `taskGroupId`, and task manipulation can be limited by
+`taskGroupId`.
```

Comment:
It does say "Each task is assigned to a task group" with the singular "a".  I should probably make it clear that taskGroupIds are slugids and not descriptive.